xbmc/cores/AudioEngine/Engines/ActiveAE/ActiveAEResamplePi.cpp

```diff
@@ -280,7 +280,7 @@ bool CActiveAEResamplePi::Init(uint64_t dst_chan_layout, int dst_channels, int d
 
   LOGTIME(3);
 
-  if (CSettings::Get().GetBool("videoplayer.usedisplayasclock") && CSettings::Get().GetInt("videoplayer.synctype") == SYNC_RESAMPLE)
+  if (CSettings::Get().GetBool("videoplayer.usedisplayasclock"))
```

Comment:
The GPU resampler stores context when resampling.
It needs to know the difference between never resampling (i.e. bypass the resample code), and resampling currently by a factor of 1.0, but may change in the future.

m_src_rate and m_dst_rate may start off as equal, but I need to know if Resample() will ever be called with a ratio != 1.0 in the future.